test/e2e/test_organization_function_run.py

```diff
@@ -0,0 +1,82 @@
+from pathlib import Path
+from typing import Any, Iterator
+
+import pytest
+
+import layer
+from layer import refresh_login
+from layer.clients import LayerClient
+from layer.contracts.accounts import Account
+from layer.contracts.fabrics import Fabric
+from layer.contracts.projects import Project
+from test.e2e.assertion_utils import E2ETestAsserter
+from test.e2e.conftest import (
+    _cleanup_project,
+    pseudo_random_account_name,
+    pseudo_random_project_name,
+)
+from test.e2e.test_function_dataset_run import (
+    test_remote_run_with_dependent_datasets_succeeds_and_registers_metadata as test_remote_dependent_datasets,
+)
+from test.e2e.test_function_model_run import (
+    test_remote_run_succeeds_and_registers_metadata as test_remote_model_train,
+)
+from test.e2e.test_layer_runtime import (
+    test_dataset_build as test_remote_layer_runtime_dataset_build,
+)
+
+
+# We use a wrapping Test class so we create only one organization
+# until we handle newly acquired permissions (after org creation)
+# better
+class TestOrganization:
+    @pytest.fixture(scope="class")
+    def initialized_organization_account(
+        self, client: LayerClient, request: Any
+    ) -> Iterator[Account]:
+        org_account_name = pseudo_random_account_name(request)
+        account = client.account.create_organization_account(org_account_name)
+
+        # We need a new token with permissions for the new org account
+        # TODO LAY-3583 LAY-3652
+        refresh_login(force=True)
```

Comment:
Is this going to cause races between different tests trying to refresh the login?

For guest logins, we had to do a per-test config.json. see https://github.com/layerai/sdk/blob/main/test/e2e/conftest.py#L148

Reply:
yes i wanted to test before going down that route, but it is likely we'll need to

Reply:
actually come to think of it: the permissions needed there are additive, the only risk i see is that we invalidate the token of other tests but other tests actually do a config refresh (without force) as part of their fixture chaining, so i think this should be okay

Reply:
this is irrelevant now that we use the pytest_sessionstart hook